code/src/components/RadioBtnOption.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+
+
+export const RadioBtnOption = ({ likeOption, onlikeOptionChange}) => {
+    const likeCheese = ["yes", "no thanks", "sometimes"];
+
+    return (
+      <div className="radiobtn-box">
+        <h2>Do you like cheese?</h2>
+        {likeCheese.map(like => (
+          <label key={like} tabIndex="0">
+            <input
+              type="radio"
+              id={like}
+              name="like"
+              value={like}
+              onChange={(event) =>  onlikeOptionChange(event.target.value)}
+              checked={likeOption === like}
```

Comment:
I saw in the comment that the blue mark didn't work here for you, I didn't write the same code as you did here (and I think your way is better?) but mine worked and I wrote it like this: 

<div className="radiobutton">
          <label className="radiobutton-css" tabIndex="0" htmlFor="february">FEB</label>
          <input
            tabIndex="0"
            id="Crazy"
            type="radio"
            checked={yourMonth.includes('Crazy')}
            onChange={() => onMonthChange('Crazy')}
          />
        </div>

And did this with every radiobutton-section. And it worked :)